pkg/swagger/swagger.go

```diff
@@ -71,12 +71,16 @@ func Run(outputDir string) error {
 
 	populateExamples(s.Paths)
 
-	err := define(s.Definitions, "github.com/Azure/ARO-RP/pkg/api/v20191231preview", "OpenShiftClusterList", "OpenShiftClusterCredentials")
+	xmsEnumList := map[string]ModelAsString{
+		"VMSize": true,
```

Comment:
Hi, @jim-minter, you are completely right. The simplest solution to this issue is to stop generating SDK for 2020-04-20 altogether. The old SDK should never change since it is already published. For the next step, we focus generation and changes only to the newly added preview API.

The alternative is changing the way how the code is generated right now, which means complete rewrite and I do believe it is not worth it.

Are you ok with simply turning the generation off for old API and SDK?